### PR TITLE
(VADC-1415 VADC-1412 VADC-1413 VADC-1456): Refactor level one headings

### PR DESCRIPTION
**Pull request by ocshawn**
Link to JIRA ticket if there is one: [VADC-1415](https://ctds-planx.atlassian.net/browse/VADC-1415) [VADC-1412](https://ctds-planx.atlassian.net/browse/VADC-1412) [VADC-1413](https://ctds-planx.atlassian.net/browse/VADC-1413) [VADC-1456](https://ctds-planx.atlassian.net/browse/VADC-1456)

### New Features
- add configurable title to Profile page

### Improvements
- update h2s to h1s for main page headings to conform with ADA standards 
  - Login page
  - Apps page
  - Individual Apps
  - Workspace page
  - Profile page
  - Resource browser

[VADC-1415]: https://ctds-planx.atlassian.net/browse/VADC-1415?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[VADC-1412]: https://ctds-planx.atlassian.net/browse/VADC-1412?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[VADC-1413]: https://ctds-planx.atlassian.net/browse/VADC-1413?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[VADC-1456]: https://ctds-planx.atlassian.net/browse/VADC-1456?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/portal_config.md [hunk 1]
```diff
@@ -714,6 +714,7 @@ Below is an example, with inline comments describing what each JSON block config
   "studyViewerConfig": [],//See docs/study_viewer.md for more details.
   "useArboristUI": false, // optional; set true to enable Arborist UI; defaults to false if absent
   "hideSubmissionIfIneligible": true, // optional; only works if Arborist UI is enabled; if set to true, link/buttons to /submission page will be hidden to users who don't have permissions to submit data; defaults to false if absent
+  "profilePageTitle": "", // title to display at top of Profile Page
```
Comment (mfshao) [inline docs/portal_config.md:717]: is this a 508 requirement too? If so maybe we should make it to have a valid default value

Comment (ocshawn) [reply docs/portal_config.md:717]: yes the requirement is to have a level 1 heading on all pages, normally meaning the page should have a H1 that lets the user know where they are. I was worried by having these by default might upset stakeholders. If you are fine with it i can make a default for this as well as the Workspace page

Comment (mfshao) [reply docs/portal_config.md:717]: yeah if that is a 508 req i think we should definitly have a default value for it
and we can pitch to our stakeholders about the change

Comment (ocshawn) [reply docs/portal_config.md]: made the updates, shows by default now

**Review by mfshao — APPROVED** [review]

**Pull request merged**